src/modules/nunjucks/DateFilter.ts

```diff
@@ -0,0 +1,27 @@
+export function dateFilter(value: string ): string {
+    // tslint:disable-next-line: prefer-const
+    let [year, month, day] = value.split('-').map(i => {
```

Comment:
That could be declared as `const`

Reply:
Ah, yes. The let is an artefact of when I was decrementing the `month` value directly, instead of simply subtracting two upon reference as I am now. I'll change to const.